openqabot/approver.py

```diff
@@ -131,6 +132,52 @@ def is_job_marked_acceptable_for_incident(self, job_id: int, inc: int) -> bool:
             pass
         return False
 
+    @lru_cache(maxsize=512)
+    def was_ok_before(self, job_id: int, inc: int) -> bool:
+        log.info("Current aggregate test %s failed. Will look at older jobs" % job_id)
+        # We need a considerable amount of older jobs, since there could be many failed manual restarts from same day
+        older_jobs = self.client.get_older_jobs(job_id, 20)
+
+        current_build = older_jobs['data'][0]['build'].replace('-1','')
+        log.info(current_build)
+        current_build_date = date(
+            int(current_build[0:4]),
+            int(current_build[4:6]),
+            int(current_build[6:8])
+            )
+        log.info("Failed job date: %s",current_build_date)
```

Comment:
I don't understand this log message. What should it tell me?

Reply:
I will remove that, was used to make sure job date and older job date were handled correctly